newIDE/app/src/MainFrame/EditorContainers/HomePage/OldGetStartedSection/index.js

```diff
@@ -0,0 +1,416 @@
+        part: 0,
+      }),
+      renderImage: props => <Unboxing {...props} />,
+    },
+    {
+      key: 'publish',
+      title: t`Improve and publish your Game`,
+      description: t`Add personality to your game and publish it online.`,
+      keyPoints: [
+        t`Game background`,
+        t`In-game obstacles`,
+        t`“You win” message`,
+        t`Sharing online`,
+      ],
+      durationInMinutes: 10,
+      // Second phase is locked if first phase is not complete
+      locked: !isTutorialPartComplete({
+        tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+        part: 0,
+      }),
+      // Phase is disabled if complete or if there's a running tutorial
+      disabled:
+        !!currentlyRunningInAppTutorial ||
+        isTutorialPartComplete({
+          tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+          part: 1,
+        }),
+      progress: getTutorialPartProgress({
+        tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+        part: 1,
+      }),
+      renderImage: props => <Building {...props} />,
+    },
+    {
+      key: 'leaderboards',
+      title: t`Add leaderboards to your online Game`,
+      description: t`Add player logins to your game and add a leaderboard.`,
+      keyPoints: [
+        t`Game personalisation`,
+        t`“Start” screen`,
+        t`Timers`,
+        t`Leaderboards`,
+      ],
+      durationInMinutes: 15,
+      // Third phase is locked if second phase is not complete
+      locked: !isTutorialPartComplete({
+        tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+        part: 1,
+      }),
+      // Phase is disabled if complete or if there's a running tutorial
+      disabled:
+        !!currentlyRunningInAppTutorial ||
+        isTutorialPartComplete({
+          tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+          part: 2,
+        }),
+      progress: getTutorialPartProgress({
+        tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+        part: 2,
+      }),
+      renderImage: props => <Podium {...props} />,
+    },
+  ];
+
+  const Subtitle = () => (
+    <ResponsiveLineStackLayout
+      justifyContent="space-between"
+      alignItems="center"
+      noColumnMargin
+      noMargin
+    >
+      <Text noMargin>
+        <Trans>Learn the basics of GDevelop and publish a first game.</Trans>
+      </Text>
+      <Checkbox
+        label={<Trans>Don't show this screen on next startup</Trans>}
+        checked={!showGetStartedSection}
+        onCheck={(e, checked) => setShowGetStartedSection(!checked)}
+      />
+    </ResponsiveLineStackLayout>
+  );
+
+  const isFlingTutorialComplete =
+    isTutorialPartComplete({
+      tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+      part: 0,
+    }) &&
+    isTutorialPartComplete({
+      tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+      part: 1,
+    }) &&
+    isTutorialPartComplete({
+      tutorialId: FLING_GAME_IN_APP_TUTORIAL_ID,
+      part: 2,
+    });
+
+  return (
+    <SectionContainer
+      title={<Trans>Guided lessons</Trans>}
```

Comment:
guided lessons will be pushed to the new recommended section I suppose now? (to ensure we don't lose them)